CONTRIBUTING.md

```diff
@@ -1 +1,109 @@
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/chunky-dev/chunky/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix. However we would request that you double check what PRs are open and whether or not another contributor is working on an issue already.
+
+### Make Changes
+
+#### Make changes in the UI
+
+Click on the pencil icon at the top right of any text file (code, markdown, etc.) to make small changes such as a typo, sentence fixes, broken links, etc. Please [create a pull request](#pull-request) for a review. 
+
+
+#### Make (more significant) changes locally
+
+IntelliJ Community Edition is recommended, get it [here](https://www.jetbrains.com/toolbox-app/)
+ - You'll want to download JetBrains Toolbox, and download `IntelliJ IDEA Community Edition` from there
+
+##### Fork the project to your account
+Go to https://github.com/chunky-dev/chunky, click on `Fork` (usually in the top right)
+
+##### Clone the project.
+with IntelliJ: `File` -> `New` -> `Project from Version Control`
+- In the url put `https://github.com/GITHUB_USERNAME/chunky`
+- You can also select a location to clone to here.
+
+with Github desktop: `Clone a repository` -> select `URL`, paste `https://github.com/GITHUB_USERNAME/chunky`
+
+with CLI: `git clone git@github.com:GITHUB_USERNAME/chunky.git` or (outdated) `git clone https://github.com/GITHUB_USERNAME/chunky`
+
+##### Open the Project
+From inside IntelliJ (if you cloned with IntelliJ you can likely skip this step)
+- `File` -> `Open` -> Select the first `build.gradle` file within the cloned folder
+
+You may have to wait some time while IntelliJ indexes the project, and imports it. (The little loading bar in the bottom right) 
+
+##### Select the correct JDK version to use to build the project
+Java 17 SDK (e.g. openjdk-17-jdk on Ubuntu):
+- `File` -> `Project Structure` -> `Project` (should be the default)
+  - If IntelliJ detected your install: select the `SDK` dropdown, select a Java 17 JDK install
+  - If not: click `Edit`, then the little `+` at the top, `Add JDK` and find your Java 17 JDK install location (select the main jdk folder).
+    - Go back to the `Project` tab, and select the newly added jdk.
+
+TODO: ADD INFO ABOUT NULLABLE AND NOTNULL ANNOTATIONS (adding them to compiler options)
+
+##### Create a new branch for your change
+In the bottom right of IntelliJ, there is a fork button, by default it will say `master`, select it, at the top select `New Branch`, and name it after what you're intending to fix. (ideally something very short, no more than a few words). eg: `fix_scene_tab_typo` or `fix-scene-tab-typo`  is fine.
```

Comment:
Isn't that button a _branch_ button then?

Reply:
Fixed; Might need to add a few screenshots so people know what button to press. Though that could be done later if needed.